sonarts-core/yarn.lock

```diff
@@ -150,13 +150,7 @@ ansi-styles@^2.2.1:
   version "2.2.1"
   resolved "https://registry.yarnpkg.com/ansi-styles/-/ansi-styles-2.2.1.tgz#b432dd3358b634cf75e1e4664368240533c1ddbe"
 
-ansi-styles@^3.0.0:
```

Comment:
@inverno why changes to this file?

Reply:
I have no clue, I just ran build.sh but back then I had other fish to fry and I wanted to see if QA had similar build problems as I was experiencing